Update animals

akezeth opened the pull request:
- Updated animals table
- Added transactions
- Added new queries

1st Transaction
![1st query](https://github.com/akezeth/Vet-clinic-database/assets/116703083/41d033ac-4a1d-48f6-8721-c7420c0cc3ea)

2nd Transaction
![2nd query](https://github.com/akezeth/Vet-clinic-database/assets/116703083/8a1072b0-6f95-48a9-9443-aef432f58623)

3rd Transaction
![3rd query](https://github.com/akezeth/Vet-clinic-database/assets/116703083/e35588ba-8c85-4820-b891-a2ea4b3a16fe)

1st query
![4th query](https://github.com/akezeth/Vet-clinic-database/assets/116703083/c9c493fa-f384-429d-991c-87dadef28ef6)

2nd query
![5th query](https://github.com/akezeth/Vet-clinic-database/assets/116703083/9d3f3f7b-3765-4f71-bba1-49b95a1cb049)

3rd query
![6th query](https://github.com/akezeth/Vet-clinic-database/assets/116703083/0dee536a-fbce-4f97-818a-b7a5405b64dd)

4th query
![7th query](https://github.com/akezeth/Vet-clinic-database/assets/116703083/046ed374-c482-4893-b075-1547f9a58152)

5th query
![8th query](https://github.com/akezeth/Vet-clinic-database/assets/116703083/d7387498-3cab-4016-bf80-6baee9c3ef53)

6th query
![9th query](https://github.com/akezeth/Vet-clinic-database/assets/116703083/45d09fc9-cdb5-4101-88a0-daf6a499f531)

akezeth committed:
Update schema.sql

akezeth committed:
Update data.sql

akezeth committed:
Update queries.sql

Abenezer-Tilahun commented on schema.sql:
```diff
@@ -6,3 +6,6 @@ create table animals (
 	weight_kg decimal
 );
+
+alter table animals 
+	add species integer;
```
- [ ] As stated in the project requirement you  should add a new column called species with **a string type** Just like the below  sample  ` ALTER TABLE animals ADD species varchar(255);`

Abenezer-Tilahun commented on data.sql:
```diff
@@ -1,5 +1,19 @@
+insert into animals 
+	(name, date_of_birth, escape_attempts, neutered, weight_kg)
+values 
+	('Charmander', '2020-02-08', 0, '0', 11),
+	('Plantmon', '2021-11-15', 2, '1', 5.7),
+	('Squirtle', '1993-04-02', 3, '0', 12.13),
+	('Angemon', '2005-06-12', 1, '1', 45),
```
- [ ] Kindly make sure that you inserted the correct value for the Animal data some of the values are negative value but so insert the value with the positive values. Kindly Fix the issues.

Abenezer-Tilahun reviewed (changes requested):
Hi @akezeth   , 

Great job for implemented  the some of the Project Requirement , but there are some changes that need to be implemented in order to proceed to the next project 👍

## Highlights 👏🏻
✔️ No Linter errors
✔️ Correct Git-Flow
✔️ Good commit messages
✔️ Professional README

![almost](https://user-images.githubusercontent.com/65860678/170491251-b1add8ba-b18f-4648-a893-20e510d957fd.gif)

## Required Changes ♻️
_Check the comments below the review._


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

akezeth committed:
Update schema.sql

akezeth commented:
- Changed the datatype of species from integer to string(varchar(20)).

tresorsawasawa commented on data.sql:
```diff
@@ -1,5 +1,19 @@
+  ('Pikachu', '2021-01-7', 1, '0', 15.04),
+  ('Devimon', '2017-05-12', 5, '1', 11);
+
+insert into animals 
+	(name, date_of_birth, escape_attempts, neutered, weight_kg)
+values 
+	('Charmander', '2020-02-08', 0, '0', 11),
+	('Plantmon', '2021-11-15', 2, '1', 5.7),
+	('Squirtle', '1993-04-02', 3, '0', 12.13),
+	('Angemon', '2005-06-12', 1, '1', 45),
+	('Boarmon', '2005-06-07', 7, '1', 20.4),
+	('Blossom', '1998-10-13', 3, '1', 17),
+	('Ditto', '2022-05-14', 4, '1', 22);
```
- [ ] As mentioned by the previous reviewer and for the project requirements, kindly ensure that  you inserte correct values for the Animal data as some values of the `weight_kg` column are negative and other are positive.
![Screenshot from 2023-07-19 16-06-18](https://github.com/akezeth/Vet-clinic-database/assets/83042742/548c6fe6-0604-4e27-9b07-17b7ded55cde)

tresorsawasawa reviewed (changes requested):
# STATUS: REQUEST CHANGES :recycle::recycle: 

Hi @akezeth 
Great work on making the changes requested by the previous reviewer  👏👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

![almost-there](https://user-images.githubusercontent.com/57562869/143511358-9cde7daf-4ad0-4db2-8f15-18b54288ff91.gif)

## To highlight :champagne: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow Gitflow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

akezeth committed:
Update data.sql

akezeth commented:
- Adjusted data.sql

I have corrected my mistakes on my pgAdmin file (as shown on the screenshot) but forget to update the schema.sql and data.sql. Sorry for the inconvenience.
![error](https://github.com/akezeth/Vet-clinic-database/assets/116703083/de5b5224-9d28-4790-a65b-26106c983396)

HAMMAS-SALEEM commented on queries.sql:
```diff
@@ -29,3 +29,72 @@ where name != 'Gabumon';
 select *
 from animals
 where weight_kg between 10.4 and 17.3;
+
+begin;
```
![image](https://github.com/akezeth/Vet-clinic-database/assets/93039359/00b70dbd-7c34-4bca-bce5-072d20d22648)

**[OPTIONAL REQUIREMENTS]**

- [ ] It would be nice if you fix your linkedin link because it is broken 😨 Kindly fix it to make your readme descriptive and professional 👍
- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍

akezeth replied:
Thanks for your comment, I have fixed it.

HAMMAS-SALEEM reviewed (approved):
# Status: Approved :green_circle: :1st_place_medal: :100:
![success](https://camo.githubusercontent.com/6ada59f6189497c1c2cae6d06f8154b7307a58643f232340e288140ed469eb28/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966)

### Hey @akezeth 👋

Great work on implementing the requested changes. Your project is complete. :blush:
There is nothing else to say about your project. :+1:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Gitflow
✔️ Following Requirements
✔️ Great work on using the transactions as expected 👍

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

akezeth committed:
Update README.md
